ONSdigital/takeon-business-layer
# Feature/spp 989/use date adjustment value for se

# Description
Updated selective editing to use the date adjusted value. Updated response query to use Date Adjusted flag. 
Story link: https://collaborate2.ons.gov.uk/jira/browse/SPP-989

# How to test?
Ran the unit  tests locally to make sure they pass.
Spun up envrionment 989, loaded 201912/202001 test data and run validation checking the logs for the correct value being used. Eg if q11 DA flag false and response used, if q21 DA flag true and adjusted response used.


# Links
Selection files used -> https://github.com/ONSdigital/spp-test-data-catalogue/tree/main/synthetic/e2e-test-v1/023-RSI
response files used -> https://github.com/ONSdigital/spp-test-data-catalogue/tree/feature/spp2052/handle-sdc-dates/synthetic/e2e-test-v1/023-RSI-v2
